README.md

```diff
@@ -140,3 +142,40 @@ This allows to delete a specific reminder.
 
 * 204 if request succeeded.
 * 404 if no reminder with this ID exists.
+
+## API notifications
+
+### POST `/notifications`
+
+Required: the header `Authorization` that identifies the user.
+
+This registers a new push notification endpoint. Returns the
+URL for this endpoint in the `Location` header.
+
+#### Input
+
+All properties must be present. There is no default value. For instance (JSON):
+```json
+{
+  "subscription": {
+    "endpoint": "...",
+    "keys": {
+      "p256dh": "<base64>",
+      "auth": "<base64>"
+    }
+  },
+  "identifier": "Chrome on Samsung G2"
+}
+```
+
+* `subscription` comes directly from browsers' [PushManager API](https://developer.mozilla.org/en-US/docs/Web/API/PushManager).
+* `identifier` is a free form string. It's intended for the end-user to distinguish between different devices/browsers.
+
+#### Output
+* 201 if request succeeded, with `location` header indicating the new resource URL.
+* 400 if some properties are missing or invalid.
+
+### GET `/notifications`, {GET,PUT,DELETE} `/notification/{id}`
+These resources behave like you would expect. Few exceptions:
+* GET don't return `auth`. This value should remain secret.
+* PUT only allows you to modify the `identifier`.
```

Comment:
What about the DELETE method is not documented.

Now PUT will have a different meaning, in theory, you can use put or post with this new endpoint to create notifications that will trigger other clients (but I think that's outside of the scope)